src/current/v23.1/import-performance-best-practices.md

```diff
@@ -50,9 +52,20 @@ This method has the added benefit of alerting on potential issues with the impor
 
 ### Import into a schema with secondary indexes
 
-When importing data into a table with secondary indexes, the import job will ingest the table data and required secondary index data concurrently. This may result in a longer import time compared to a table without secondary indexes. However, this typically adds less time to the initial import than following it with a separate pass to add the indexes. As a result, importing tables with their secondary indexes is the default workflow, suitable for most import jobs.
+When importing data into a table with secondary indexes, the import job will ingest the table data and required secondary index data concurrently. This may result in a longer import time compared to a table without secondary indexes. However, this typically adds less time to the initial import than following it with a separate pass to add the indexes. As a result, importing tables with their secondary indexes is the default workflow, an effective strategy for most migrations.
+
+However, in **large** imports, it may be preferable to temporarily remove the secondary indexes from the schema, perform the import, and then re-create the indexes separately. Write operations on tables with many secondary indexes take longer to complete, and importing large datasets have a greater risk of timeouts. Removing the table's secondary indexes allows you to separate the initial data import from the secondary index creation operation, and the total import time is lower. Separating these operations also provides increased visibility into each operation's progress, and the ability to retry each operation independently if you encounter errors or timeouts.
 
-However, in **large** imports, it may be preferable to temporarily remove the secondary indexes from the schema, perform the import, and then re-create the indexes separately. This provides increased visibility into its progress and ability to retry each step independently.
+When recreating the secondary indexes, execute all the [`CREATE INDEX`](create-index.html) statements in a single transaction. For example:
```

Comment:
nit: explain the benefit of doing this

Reply:
I've done some testing around this and found that, particularly with larger data sets, creating the indexes serially was faster than creating them all in a single transaction.

Reply:
Ok, I've reworked the section to say that executing individual statements are faster. Is it important to describe why?